### PR TITLE
Update diagnose ecs tcr methods

### PR DESCRIPTION
**Pull request by znicholls**
# Pull request

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Documentation added (where applicable)
- [x] Example added (either to an existing notebook or as a new notebook, where applicable)
- [x] Description in ``CHANGELOG.rst`` added

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <http://link-to-pr.com>`_) Added feature which does something
- (`#XX <http://link-to-pr.com>`_) Fixed bug identified in (`#XX <http://link-to-issue.com>`_)
```

depends on #283

### CODE REVIEW

File: pymagicc/core.py [hunk 1]
```diff
@@ -1069,166 +1142,268 @@ def diagnose_tcr_ecs_tcre(self, **kwargs):
...
         )
         # drop all the irrelevant inverse emissions
-        timeseries = timeseries.filter(variable="Inverse Emissions*", level=1, keep=False)
-        tcr, ecs, tcre = self.get_tcr_ecs_tcre_from_diagnosis_results(timeseries)
-        return {"tcr": tcr, "ecs": ecs, "tcre": tcre, "timeseries": timeseries}
+        timeseries = timeseries.filter(
+            variable="Inverse Emissions*", level=1, keep=False
+        )
+        # drop the final year as concs stay constant from some reason,
+        # MAGICC bug...
```
Comment (lewisjared) [inline pymagicc/core.py]: Does this have a issue in the MAGICC repo?

File: pymagicc/MAGICC6/run/TCRECS_CO2_CONC.IN [hunk 2]
```diff
@@ -1,3415 +0,0 @@
-TCR Diagnosis input file
-Written by: diagnose-tcr notebook in pymagicc package
-
-&THISFILE_SPECIFICATIONS
-    THISFILE_DATACOLUMNS = 1
```
Comment (lewisjared) [inline pymagicc/MAGICC6/run/TCRECS_CO2_CONC.IN]: Can the TCRECS_*_CONC.IN from magicc7 also be deleted?

File: pymagicc/core.py (hunk 1 above)
Comment (znicholls) [reply pymagicc/core.py]: No, probably should (was trying to avoid an extra distraction..)

File: pymagicc/MAGICC6/run/TCRECS_CO2_CONC.IN (hunk 2 above)
Comment (znicholls) [reply pymagicc/MAGICC6/run/TCRECS_CO2_CONC.IN]: yep

**Pull request merged**